docs: IA Phase 10 (P2): add "Measure impact / Reporting & ROI" page (#733)
Promptless/docs: src/content/docs/docs/measure/reporting-and-roi.mdx

Reviewer: Update "git host" to "git history" globally.

---
title: Measure the impact of Promptless
description: >-
  Track the value of keeping docs current with Promptless using the signals the
  product surfaces today, and turn them into a case you can take to leadership
slug: docs/measure/reporting-and-roi
sidebar:
  hidden: false
  order: 45
  label: Reporting & ROI
---
import { Aside } from '@astrojs/starlight/components';

> When leadership asks what the docs function is worth, you need numbers you can defend. If you own Promptless, much of that story already lives in the product and in your Git host: doc updates drafted, reviewed, and merged, plus the changes that no longer wait on someone finding the time. This page shows which signals to track, where to read each one, and how to assemble them into a case for spend, headcount, or the role itself. If you have no manager to report to—say you maintain an open-source project—the same numbers make the case to a foundation, a sponsor, or your community, or simply justify the hours you put in.

Reviewer: ```suggestion
When leadership asks what the docs function is worth, you need numbers you can defend. If you use Promptless, much of that story already lives in the product and in your Git history: doc updates drafted, reviewed, and merged, plus the changes that no longer wait on someone finding the time. This page shows which signals to track, where to read each one, and how to assemble them into a case for spend or headcount. If you have no manager to report to—say you maintain an open-source project—the same numbers make the case to a foundation, a sponsor, or your community, or simply justify the hours you put in.
```

<Aside type="note">

> Promptless does not ship a built-in reporting dashboard or a metrics export. The signals below come from surfaces you already use—the Suggestions list, the Triggers page, Change History, and your Git host—so you assemble the report yourself. Track the handful of numbers that matter to your leadership rather than everything you could count.

Reviewer: ```suggestion
Promptless does not ship a built-in reporting dashboard or a metrics export. The signals below come from surfaces you already use, so you assemble the report yourself. Track the handful of numbers that matter to your leadership rather than everything you could possibly count.
```

</Aside>

## What you can observe today

> Four surfaces carry the raw signal. Each answers a different question about the value Promptless is adding.

Reviewer: ```suggestion
Four surfaces carry raw signal, and each answers a different question about the value Promptless is adding.
```

- **Suggestions list.** Every draft Promptless creates, with status buckets—**Ready**, **PR Open**, and **Closed**—and a per-suggestion review-time estimate. This is your view of drafting volume and review load. Filter by **Docs location** to attribute activity to a specific collection, or by **Trigger source** to see which signals produce the most drafts.
- **Triggers page.** Every event Promptless reacted to over the recent window, each marked **Suggestion created** or **No suggestion created**. This shows coverage—how many of your product's changes Promptless caught—and, through the no-suggestion outcomes, how well it is tuned to skip changes that don't need docs. Search reaches events older than the default window, and the **From** and **To** date fields scope a specific range.

> - **Change History.** A filterable record of the suggestions Promptless has produced, including by trigger source. Use it to pull activity for a specific period or origin when you build a report.

Reviewer: @promptless That doesn't exist. Update references to that url in all locations.

> - **Your Git host.** Merged documentation PRs are the ground truth for shipped work. A merged Promptless PR is a doc change that went live; the PR's author, reviewers, and timestamps live in GitHub, GitLab, or Bitbucket, where your team already reviews them. Promptless opens these PRs under its own account, so you can filter to that author to isolate its contribution—read the exact account name off any docs PR Promptless has opened in your repo.

Reviewer: ```suggestion
- **Your Git history.** Merged documentation PRs are the ground truth for shipped work. A merged Promptless PR is a doc change that went live; the PR's author, reviewers, and timestamps live in GitHub, GitLab, or Bitbucket, where your team already reviews them. Promptless opens these PRs under its own account, so you can filter to that author to isolate its contribution—read the exact account name off any docs PR Promptless has opened in your repo.
```

---
title: Measure the impact of Promptless
description: >-
  Track the value of keeping docs current with Promptless using the signals the
  product surfaces today, and turn them into a case you can take to leadership
slug: docs/measure/reporting-and-roi
type: guide
tags:
  - measure
  - guide
timestamp: "2026-07-17T20:03:00Z"
sidebar:
  hidden: false
  order: 45
  label: Reporting & ROI
---

import { Aside } from '@astrojs/starlight/components';

> When leadership asks what the docs function is worth, you need numbers you can defend. If you use Promptless, much of that story already lives in the product and in your Git history: doc updates drafted, reviewed, and merged, plus the changes that no longer wait on someone finding the time. This page shows which signals to track, where to read each one, and how to assemble them into a case for spend or headcount. If you have no manager to report to—say you maintain an open-source project—the same numbers make the case to a foundation, a sponsor, or your community, or simply justify the hours you put in.

Reviewer: ```suggestion
When leadership asks what the Docs function is worth, you need numbers you can defend. If you use Promptless, much of that story already lives in the product and in your Git history: doc updates drafted, reviewed, and merged, plus the changes that no longer wait on someone finding the time. This page shows which signals to track, where to read each one, and how to assemble them into a case for spend or headcount. If you have no manager to report to—say you maintain an open-source project—the same numbers make the case to a foundation, a sponsor, or your community, or simply justify the hours you put in.
```

- **Review throughput and load.** Use the Suggestions list buckets and review-time estimates to show how much review the queue represents and how quickly your team clears it—the effort Promptless converts from writing into reviewing.
- **Time saved.** Multiply doc updates shipped by your team's own estimate of the hours a comparable manual update takes. Promptless doesn't measure this for you, so state your assumption plainly; a defensible estimate lands better than a precise-looking number no one can source.

<Aside type="caution">
  These counts describe activity and output, not documentation quality. Pair them with the outcome signals in the next section, so the story is about outcomes, not just volume.
</Aside>

## Pair output with outcomes

> The metrics above count what Promptless produces. To show whether that output actually helped readers or reduced cost, pair it with outcome signals from tools your team already runs. Promptless does not produce, measure, or export any of these outcome numbers—they live in your own analytics, support, and docs-platform tools, and you read them there. The same signals point to what to improve next, not just what to report.

Reviewer: ```suggestion
The metrics above count what Promptless produces. To show whether that output actually helped readers or reduced cost, pair it with outcome signals from tools your team already runs. Promptless does not produce, measure, or export any of these outcome numbers. Instead, they live in your own analytics, support, and docs-platform tools, and you read them there. The same signals point to what to improve next, not just what to report.
```

> - **Support-ticket deflection / self-service rate.** This is the strongest dollar lever and the clearest before/after comparison. Read it from your helpdesk or support platform (such as Zendesk or Intercom, or a free channel like GitHub Issues or Discussions if that is where your users ask questions). After a Promptless-driven update to a documented topic, compare ticket volume on that topic before and after; the value is tickets deflected multiplied by your support team's fully-loaded cost per ticket, a figure support ops usually already has. Attribution here is an estimate, so show it as before/after on a specific topic.

Reviewer: ```suggestion
- **Support-ticket deflection / self-service rate.** This is the strongest dollar lever and the clearest before/after comparison. Read it from your helpdesk or support platform (such as Zendesk or Intercom, or a free channel like GitHub Issues or Discussions if that is where your users ask questions). After a Promptless-driven update to a documented topic, compare ticket volume on that topic before and after. The value is tickets deflected multiplied by your support team's fully-loaded cost per ticket, a figure support ops usually already has. Attribution here is an estimate, so show it as before/after on a specific topic.
```

- **"Was this helpful?" ratings and feedback comments.** This is a per-page reader verdict, directly attributable to a specific updated page, so it is ideal for before/after on a Promptless-driven change. It is often built into your docs platform (such as Mintlify, GitBook, or ReadMe, or similar) or added with a feedback widget. This signal is self-selected and skews negative, so read it as a trend and a triage signal, not an absolute score.
- **Site-search "no-results" rate and top queries.** This is the highest-signal content-gap map, because users describe in their own words what they cannot find. Read it from your site-search analytics (such as Algolia, a docs platform's built-in search dashboard, or similar). Recurring no-result or high-volume queries tell you which gaps to close next and give you language to match, and a falling no-results rate on a topic after an update is a reader-side outcome.

> - **Page views and engagement on changed pages.** Confirms reach—that the audience for an updated page actually exists. Read it from your web or product analytics (such as GA4, Plausible, or similar). Traffic is not success: a high bounce can be fine on a page that answers a question fast, so read engagement (such as GA4's engaged sessions or average engagement time) alongside raw views, and treat traffic as context, not proof.

Reviewer: ```suggestion
- **Page views and engagement on changed pages.** Confirms reach—that the audience for an updated page actually exists. Read it from your web or product analytics (such as GA4, Plausible, or similar). Traffic is not success. A high bounce can be fine on a page that answers a question fast, so read engagement (such as GA4's engaged sessions or average engagement time) alongside raw views, and treat traffic as context, not proof.
```

3. **Frame output as leverage.** Lead with doc updates shipped and share of doc changes automated—the numbers that show the docs function keeping pace with the product. Add time saved with your stated per-update assumption.
4. **Attach an outcome signal.** Pull one or two outcome metrics from your own tools—ticket deflection, helpfulness ratings, or a search-gap trend—for the same collections and window so the report speaks to reader experience, not just throughput. Where you have the inputs, convert them to a dollar figure using the formulas above.
5. **Keep the sources visible.** Note which surface each number came from. Leadership trusts a figure it can trace back to the Suggestions list, the Triggers page, or a merged PR far more than a bare total.

<Aside type="note">
  Learn your way around the surfaces this page draws on: the Suggestions list and Triggers page in [Use the web interface](/docs/work-the-queue/web-interface), and where lasting context lives in the [Agent Knowledge Base](/docs/get-the-most-out/agent-knowledge-base).
</Aside>

> Want help shaping a report for your leadership? Contact us at [help@gopromptless.ai](mailto\:help@gopromptless.ai).

Reviewer: ```suggestion
```
